diff --git a/node_modules/webuploader/dist/webuploader.fis.js b/node_modules/webuploader/dist/webuploader.fis.js
index 2349319..050215c 100644
--- a/node_modules/webuploader/dist/webuploader.fis.js
+++ b/node_modules/webuploader/dist/webuploader.fis.js
@@ -4915,24 +4915,24 @@ module.exports = (function( root, factory ) {
                 mouseHandler = function( e ) {
                     owner.trigger( e.type );
                 };
+                function onChange( e ) {
+                  var fn = onChange,
+                      clone;

-                input.on( 'change', function( e ) {
-                    var fn = arguments.callee,
-                        clone;
+                  me.files = e.target.files;

-                    me.files = e.target.files;
+                  // reset input
+                  clone = this.cloneNode( true );
+                  clone.value = null;
+                  this.parentNode.replaceChild( clone, this );

-                    // reset input
-                    clone = this.cloneNode( true );
-                    clone.value = null;
-                    this.parentNode.replaceChild( clone, this );
+                  input.off();
+                  input = $( clone ).on( 'change', fn )
+                          .on( 'mouseenter mouseleave', mouseHandler );

-                    input.off();
-                    input = $( clone ).on( 'change', fn )
-                            .on( 'mouseenter mouseleave', mouseHandler );
-
-                    owner.trigger('change');
-                });
+                  owner.trigger('change');
+              }
+                input.on( 'change', onChange);

                 label.on( 'mouseenter mouseleave', mouseHandler );

diff --git a/node_modules/webuploader/src/runtime/html5/filepicker.js b/node_modules/webuploader/src/runtime/html5/filepicker.js
index f373770..cb58fb6 100644
--- a/node_modules/webuploader/src/runtime/html5/filepicker.js
+++ b/node_modules/webuploader/src/runtime/html5/filepicker.js
@@ -59,24 +59,24 @@ define([
             mouseHandler = function( e ) {
                 owner.trigger( e.type );
             };
+            function onChange( e ) {
+              var fn = onChange,
+                  clone;

-            input.on( 'change', function( e ) {
-                var fn = arguments.callee,
-                    clone;
+              me.files = e.target.files;

-                me.files = e.target.files;
+              // reset input
+              clone = this.cloneNode( true );
+              clone.value = null;
+              this.parentNode.replaceChild( clone, this );

-                // reset input
-                clone = this.cloneNode( true );
-                clone.value = null;
-                this.parentNode.replaceChild( clone, this );
+              input.off();
+              input = $( clone ).on( 'change', fn )
+                      .on( 'mouseenter mouseleave', mouseHandler );

-                input.off();
-                input = $( clone ).on( 'change', fn )
-                        .on( 'mouseenter mouseleave', mouseHandler );
-
-                owner.trigger('change');
-            });
+              owner.trigger('change');
+          }
+            input.on( 'change', onChange);

             label.on( 'mouseenter mouseleave', mouseHandler );

